llvm/lib/Target/NVPTX/NVPTXCtorDtorLowering.cpp

```diff
@@ -32,6 +33,11 @@ static cl::opt<std::string>
               cl::desc("Override unique ID of ctor/dtor globals."),
               cl::init(""), cl::Hidden);
 
+static cl::opt<bool>
+    CreateKernels("nvptx-lower-global-ctor-dtor-kernel",
```

Comment:
Nit: That's a very long option name. Perhaps we could generalize and shorten it down to just `nvptx-global-init` ?

Reply:
Works for me.